08 Modules/01 Creating Modules.py

```diff
@@ -16,8 +16,9 @@
 # To import functions from one module we can use two ways.
 
 from sales import calc_tax, calc_shipping # In here we select the module and then separated by comma each of the functions.
+
 # Some tutorials teach to import multiple functions using an "*" "from sales import *".
-# This is a bad practice because if we import everything blinding, some of our functions or variables in our "sales.py" module may override some objects with the same name in the current module.
+from sales import *  # This is a bad practice because if we import everything blinding, some of our functions or variables in our "sales.py" module may override some objects with the same name in the current module.
```

Comment:
Why are you adding here the import statement?

It is already in the previous line.